tests/foreman/cli/test_hammer.py

```diff
@@ -12,28 +12,25 @@
 
 :TestType: Functional
 
-:CaseImportance: High
+:CaseImportance: Critical
 
 :Upstream: No
 """
 import io
 import json
 import re
+import time
 
 import pytest
-from fauxfactory import gen_string
 
 from robottelo.cli import hammer
-from robottelo.cli.admin import Admin
-from robottelo.cli.defaults import Defaults
-from robottelo.cli.factory import make_org
-from robottelo.cli.factory import make_product
 from robottelo.constants import DataFile
+from robottelo.logging import logger
 
 
-HAMMER_COMMANDS = json.loads(DataFile.HAMMER_COMMANDS_JSON.read_bytes())
+HAMMER_COMMANDS = json.loads(DataFile.HAMMER_COMMANDS_JSON.read_text())
 
-pytestmark = [pytest.mark.tier1]
+pytestmark = [pytest.mark.tier1, pytest.mark.upgrade]
```

Comment:
add e2e as well ?

Reply:
There are not e2e tests, still can add?

Reply:
@jyejare Updated

Reply:
imho `test_positive_all_options` falls into upgrade/e2e category, but for other tests in the module I'm not convinced

Reply:
@pondrejk As this are all basic hammer scenarios, so i think its fine to include those into upgrade alltier and e2e runs?